databases/week2/part3.sql

```diff
@@ -0,0 +1,15 @@
+USE hyf_class2023;
+-- Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT task.id, task.title, user.email FROM task INNER JOIN user ON task.user_id = user.id WHERE user.email LIKE '%@spotify.com';
+-- Get all the tasks for 'Donald Duck' with status 'Not started'
+SELECT task.id, task.title, task.created, user.name FROM task
+INNER JOIN status ON task.status_id = status.id
+INNER JOIN user ON task.user_id = user.id
+WHERE status.name='Not started' AND user.name='Donald Duck';
```

Comment:
Please compare your & see the diff
```
SELECT task.id, task.title, task.created, user.name 
FROM task
join user_task on task.id=user_task.task_id
JOIN status ON task.status_id = status.id 
JOIN user ON user_task.user_id = user.id
WHERE status.name='Not started' AND user.name='Donald Duck';
```

Reply:
it is my bad, I worked with the table I created for lesson 1 where I did not have user_task table